examples/bootstrap-orm.php

```diff
@@ -24,7 +25,7 @@
 
     $entityManager = EntityManager::create(
         [
-            'driverClass' => \Doctrine\DBAL\Driver\PDOSqlite\Driver::class,
+            'driverClass' => \Doctrine\DBAL\Driver\PDO\SQLite\Driver::class,
```

Comment:
I think we need to require a newer `doctrine/dbal` version, at least in `require-dev`